contracts/base/BaseLiquidityManagement.sol

```diff
@@ -34,17 +37,27 @@ contract BaseLiquidityManagement is IBaseLiquidityManagement, SafeCallback {
     using TransientStateLibrary for IPoolManager;
     using SafeCast for uint256;
     using LiquiditySaltLibrary for IHooks;
+    using PositionLibrary for IBaseLiquidityManagement.Position;
 
     mapping(address owner => mapping(LiquidityRangeId rangeId => Position)) public positions;
 
     constructor(IPoolManager _manager) ImmutableState(_manager) {}
 
-    function zeroOut(BalanceDelta delta, Currency currency0, Currency currency1, address owner, bool claims) public {
-        if (delta.amount0() < 0) currency0.settle(manager, owner, uint256(int256(-delta.amount0())), claims);
-        else if (delta.amount0() > 0) currency0.send(manager, owner, uint128(delta.amount0()), claims);
+    function _closeCallerDeltas(
+        BalanceDelta callerDeltas,
+        Currency currency0,
+        Currency currency1,
+        address owner,
+        bool claims
+    ) internal {
+        int128 callerDelta0 = callerDeltas.amount0();
+        int128 callerDelta1 = callerDeltas.amount1();
 
-        if (delta.amount1() < 0) currency1.settle(manager, owner, uint256(int256(-delta.amount1())), claims);
-        else if (delta.amount1() > 0) currency1.send(manager, owner, uint128(delta.amount1()), claims);
+        if (callerDelta0 < 0) currency0.settle(manager, owner, uint256(int256(-callerDelta0)), claims);
+        else if (callerDelta0 > 0) currency0.send(manager, owner, uint128(callerDelta0), claims);
+
+        if (callerDelta1 < 0) currency1.settle(manager, owner, uint256(int256(-callerDelta1)), claims);
+        else if (callerDelta1 > 0) currency1.send(manager, owner, uint128(callerDelta1), claims);
```

Comment:
we should in-line a comment saying that on liquidity-increase, we _never_ send any positive delta, and instead is collected to `position.tokensOwed` (for callerDelta > 0)

Reply:
thats a great idea